<?xml version="1.0"?>    
<launch>    

    <!--<param name="/use_sim_time" value="true" /> -->

    <!--********************load pcd in rviz*********************-->
     <!--<node pkg="quadruped_simshow" type="load_legoloam_map" name="load_legoloam_map" output="screen">
        <param name="global_map_file" value="/home/hitnuc/0-sim_bag/map/dog1.pcd"/>
    </node> -->
    
    <!-- <node pkg="quadruped_simshow" type="load_raw_map" name="load_raw_map" output="screen">
        <param name="global_map_file" value="/home/hitnuc/0-sim_bag/map/dog1.pcd"/>
    </node> -->

    <!--<node pkg="quadruped_simshow" type="loadmap" name="loadmap" output="screen">
        <param name="global_map_file" value="/home/hitnuc/0-sim_bag/map/dog1.pcd"/>
        <param name="filter_point_num" value="300000"/>
        <param name="voxel_grid_filter" value="0.02"/>
    </node>-->

    <!--show path in rviz-->
    <node pkg="quadruped_simshow" type="quadruped_util" name="quadruped_util" output="screen"/>

    <!--show-->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find quadruped_simshow)/rviz/sim_localization.rviz"/>

    <!--<node name="tf_map2odom" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0  /map /odom 50"/>-->
    <!--<node name="tf_base2odom" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0  /odom /base_link 50"/>-->

</launch>